doc/StaticWorkflow.md

```diff
@@ -0,0 +1,275 @@
+# Preferences Edge Proxy Refinement I
+
+This describes the requests, responses, and payloads for the workflow where a
+static site makes save/retrieve requests of a secured preferences server.  For
+example, the static site may have incorporated UIO+ into their pages.  The user
+wants to save or retrieve their UIO+ preferences.
```

Comment:
I believe you mean to say UIO rather than UIO+. UIO+ is the chrome extension, which at some point would also make sense to connect to the prefs server.

Reply:
Right.  Done.